showcase/app/styles/showcase-pages/advanced-table.scss

```diff
@@ -22,17 +22,13 @@ body.components-advanced-table {
     .shw-component-advanced-table-cell-content-div {
       display: flex;
       align-items: center;
-  
+
       .hds-icon {
         flex: none;
         margin-right: 5px;
       }
     }
 
-  .shw-component-advanced-table-fixed-height-wrapper {
```

Comment:
[todo] see comment above, about wrapping `<div>` that can be removed

Reply:
Removed!